src/git/install.sh

```diff
@@ -26,6 +26,8 @@ fi
 # Get an adjusted ID independent of distro variants
 if [ "${ID}" = "debian" ] || [ "${ID_LIKE}" = "debian" ]; then
     ADJUSTED_ID="debian"
```

Comment:
Can we add tests for validating the changes? thanks! 

Refer to https://github.com/devcontainers/features/blob/4407a4d98381b74890edc3bfabcadbb48f7873c1/test/git/scenarios.json

Reply:
Done. Added test for system install on Alpine